accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,102 @@
+- Feature Name: CaaS Platform support
+- Start Date: 2019-09-24
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Initial integration for managing a CaaS Platform cluster from Uyuni/SUSE Manager.
+
+# Motivation
+[motivation]: #motivation
+
+Uyuni/SUSE Manager is the systems management tool and must offer to the end-user the possibility to manage other SUSE products like CaaS Platform.
+Uyuni/SUSE Manager already supports Kubernetes cluster integration. It works with CaaS Platform version 3 (container engine: Docker).
+
+In this regard, this RFC will cover the following aspects:
+
+- Node status reporting
+- Update the current Kubernetes integration of Uyuni/SUSE Manager to work with CaaS Platform version 4 while still supporting CaaS Platform version 3
+- Document how to deploy CaaS Platform using AutoYaST
```

Comment:
This should be three different steps/iterations instead of one. :-/

Reply:
> This should be three different steps/iterations instead of one. :-/

Do you mean shipping those 3 features in 3 different MUs? Why?

Or implementing this in 3 steps, that may or may not ship in the same MU?

Reply:
Implement iteratively, ship early, ship often. Each step above represent a product value increment which you can get customer feedback about.